internal/codegen/tpl_stencil.go

```diff
@@ -181,9 +182,11 @@ func (s *TplStencil) ReadFile(name string) (string, error) {
 	return string(b), nil
 }
 
-type ReadDirEntry struct {
-	Name  string
-	IsDir bool
+// ReadDirEntry is a partial of [os.DirEntry] returned by
+// [TplStencil.ReadDir]
+type ReadDirEntry interface {
```

Comment:
Does this work as a field of readdirentry from inside the template or is this another breaking change since you have to call a function now?

Reply:
It automatically calls the function :D. `stencil.` is a function, for example, but it feels like a static field